projectOne/src/oneTwo.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+public class oneTwo {
+    public static void main(String[] args) {
+        System.out.println("Введите номер дня недели (1-7)");
+        Scanner scanner = new Scanner(System.in);
+        int dayNumber = scanner.nextInt();
+        switch (dayNumber) {
+            case 1:
+                System.out.println("Понедельник");
+                break;
+            case 2:
+                System.out.println("Вторник");
+                break;
+            case 3:
+                System.out.println("Среда");
+                break;
+            case 4:
+                System.out.println("Четверг");
+                break;
+            case 5:
+                System.out.println("Пятница");
+                break;
+            case 6:
+                System.out.println("Суббота");
+                break;
+            case 7:
+                System.out.println("Воскресенье");
+                break;
+        }
```

Comment:
Java 8 contains DateTime API and it offers wide functionality for date manipulation. In this case you can use DayOfWeek.of(dayNumber) instead of a switch block.